src/main/java/seedu/address/logic/commands/SortCommand.java

```diff
@@ -0,0 +1,31 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.model.Model;
+import seedu.address.model.person.PersonContainsTagPredicate;
+
+/**
+ * Lists persons that match all tags specified by the user.
```

Comment:
"persons" could be changed to "classmates", "user" to "student" 🤔

Reply:
Ok, will change and push soon!